app/src/main/java/org/mozilla/reference/browser/ext/Context.kt

```diff
@@ -23,3 +31,27 @@ val Context.components: Components
 
 fun Context.getPreferenceKey(@StringRes resourceId: Int): String =
     resources.getString(resourceId)
+
+/**
+ *  Shares content via [ACTION_SEND] intent.
+ *
+ * @param text the data to be shared  [EXTRA_TEXT]
+ * @param subject of the intent [EXTRA_TEXT]
+ * @return true it is able to share false otherwise.
+ */
+fun Context.share(text: String, subject: String = ""): Boolean {
+    return try {
+        Intent(ACTION_SEND).apply {
+            type = "text/plain"
+            putExtra(EXTRA_SUBJECT, subject)
+            putExtra(EXTRA_TEXT, text)
+            flags = FLAG_ACTIVITY_NEW_TASK
+        }.also { intent ->
+            startActivity(intent)
+        }
+        true
+    } catch (e: ActivityNotFoundException) {
+        Log.log(ERROR, "Reference-Browser")
```

Comment:
You may want to use `Logger.error()`. However this doesn't look like a severe error and a perfectly normal case. So maybe warning/info is enough? :)

Reply:
Let's go with warning!